src/app/api/projects/route.tsx

```diff
@@ -11,14 +12,21 @@ export async function GET(req: NextRequest) {
     const address = searchParams.get("address");
     const logs = await ethGetLogs(address as Address);
     const attestations = await getAttestations(logs);
-    const serializedAttestations = attestations.map((attestation: any) => ({
+    let serializedAttestations = attestations.map((attestation: any) => ({
       ...attestation,
-      result: attestation.result.map((value: any) =>
-        typeof value === "bigint" ? value.toString() : value
-      ),
+      result: attestation.result.map((value: any) => {
+        return typeof value === "bigint" ? value.toString() : value;
+      }),
```

Comment:
redundant return right?
```suggestion
      result: attestation.result.map((value: any) => 
        typeof value === "bigint" ? value.toString() : value;
      ),
```

Reply:
same results with slightly less code